.eslintrc

```diff
@@ -8,7 +8,7 @@
     "curly": 0,
     "comma-dangle": 0,
     "comma-spacing": 0,
-    "eqeqeq": 0,
+    "eqeqeq": 2,
```

Comment:
According to the eslint docs the rule produces warnings. I did not see a section that described what violations were considered errors. Safer to elevate the rule to error on any violation.

If the community feels a particular case (or cases) should be allowed, we can make the necessary changes to this rule.